Application/EdFi.Ods.Security/AuthorizationStrategies/Relationships/Filters/RelationshipsAuthorizationFilters.cs

```diff
@@ -21,6 +21,9 @@ public static class RelationshipsAuthorizationFilters
         private static readonly Lazy<FilterApplicationDetails> _communityProviderIdToCommunityProviderId =
             new Lazy<FilterApplicationDetails>(() => CreateClaimValuePropertyFilter("CommunityProviderId"));
 
+        private static readonly Lazy<FilterApplicationDetails> _organizationDepartmentIdToOrganizationDepartmentId =
```

Comment:
This should be removed because we are not planning to support API clients assigned directly to OrganizationDepartments at this point.

Reply:
Fixed it . Thanks